### ISSUE TITLE
Sklearn Integration

### ISSUE THREAD

**Issue by gaurav274** [georgia-tech-db/evadb#960]
### Search before asking

- [X] I have searched the EvaDB [issues](https://github.com/georgia-tech-db/eva/issues) and found no similar feature requests.

### Description

Expand our integrations with ML libraries.

- [x] Sklearn Linear Model support #1162 
- [x] Sklearn documentation under [AI engines](https://evadb.readthedocs.io/en/stable/source/reference/ai/model-train.html) #999 
- [ ] Use [auto-sklearn](https://automl.github.io/auto-sklearn/master/) for automl.

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by jineetd** [comment]
I would like to pick up this issue and contribute to the integration.

**Comment by xzdandy** [comment]
Hi @jineetd, thanks for bringing in the linear model support from Sklearn to EvaDB. I will take care of the documentation. You can think about ideas how we can scale the support to other models from Sklearn. Thanks!

**Comment by xzdandy** [comment]
The integration test failed on circle ci. @jineetd Please take a look. 
```
self = <test.integration_tests.long.test_model_train.ModelTrainTests testMethod=test_sklearn_regression>

    def test_sklearn_regression(self):
        create_predict_function = """
            CREATE FUNCTION IF NOT EXISTS PredictHouseRent FROM
            ( SELECT number_of_rooms, number_of_bathrooms, days_on_market, rental_price FROM HomeRentals )
            TYPE Sklearn
            PREDICT 'rental_price';
        """
        execute_query_fetch_all(self.evadb, create_predict_function)
    
        predict_query = """
            SELECT PredictHouseRent(number_of_rooms, number_of_bathrooms, days_on_market, rental_price) FROM HomeRentals LIMIT 10;
        """
>       result = execute_query_fetch_all(self.evadb, predict_query)

test/integration_tests/long/test_model_train.py:87: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
evadb/server/command_handler.py:93: in execute_query_fetch_all
    res_batch = execute_query(
evadb/server/command_handler.py:72: in execute_query
    res_batch = execute_statement(
evadb/server/command_handler.py:53: in execute_statement
    batch_list = list(output)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <evadb.executor.plan_executor.PlanExecutor object at 0x7fe4bf7db910>
do_not_raise_exceptions = False, do_not_print_exceptions = False

    def execute_plan(
        self,
        do_not_raise_exceptions: bool = False,
        do_not_print_exceptions: bool = False,
    ) -> Iterator[Batch]:
        """execute the plan tree"""
        try:
            execution_tree = self._build_execution_tree(self._plan)
            output = execution_tree.exec()
            if output is not None:
                yield from output
        except Exception as e:
            if do_not_raise_exceptions is False:
                if do_not_print_exceptions is False:
                    logger.exception(str(e))
>               raise ExecutorError(e)
E               evadb.executor.executor_utils.ExecutorError: 'sqft'

evadb/executor/plan_executor.py:183: ExecutorError
```
[Circle CI](https://app.circleci.com/pipelines/github/georgia-tech-db/evadb/5540/workflows/583cc3d7-6283-4527-bcdc-4ed4fd6ef6ef/jobs/39840)

**Comment by xzdandy** [comment]
The above error will be fixed in #1209

**Comment by xzdandy** [comment]
auto-sklearn does not work python3.10 

So we will drop the auto-sklearn integration for now.  

Reference: https://github.com/automl/auto-sklearn/issues/1694
